video/out/vo.c

```diff
@@ -764,7 +764,7 @@ void vo_wakeup(struct vo *vo)
 static bool still_displaying(struct vo *vo)
 {
     struct vo_internal *in = vo->in;
-    int64_t now = mp_time_us();
+    int64_t now = mp_time_ns();
```

Comment:
Woops missed this one. Is this right? Shouldn't duration and so on still be in microseconds?

Reply:
Looks like a stray change indeed, it is not even related to anything I was working on. Fixed.